config/application.yml

```diff
@@ -0,0 +1,11 @@
+# Add configuration values here, as shown below.
```

Comment:
open .gitignore file and add config/application.yml
Then run git rm --cached config/application.yml to untrack this file. Finally, commit and push again